src/components/Styles/Bookcard.css

```diff
@@ -5,8 +5,16 @@
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  width: 100vw;
   padding: 2rem 6rem;
+  margin: 1.5rem auto 1rem auto;
+  background-color: #fff;
+  width: 90%;
+  border-radius: 2px;
+}
+
+.bookdetails {
+  margin: 0;
+  padding: 0;
```

Comment:
- [ ] Good job working on the styling, but there little discrepancies that need to be taken care of. Please kindly compare the images below, you would notice that the font-family, font-weight, font-size and color of most of the text is different. Please kindly fix this.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/230638035-876d6ff7-0cdc-47e4-bd9a-950070d4e73a.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/230638001-74e7d127-47d8-4518-9e0e-48249ba3e57e.png)